main.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+function makeSortable (elem) {
+  var countClick = sessionStorage.getItem("countClick");
+   
+  let append = document.createElement('p');
```

Comment:
append is not a good name for DOM element variable, it can be name for a function